requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+# formatting
+yapf==0.32.0
```

Comment:
remove this file, add it to setup.py https://github.com/alpa-projects/alpa/blob/main/setup.py#L89